stories/organisms/SelectedItemBadge.stories.tsx

```diff
@@ -489,6 +489,7 @@ export const SelectedItemBadgeWithSingleSelectItems: Story = {
                             }}
                             key={radius}
                             value={radius}
+                            isSelected={selectedRadius === radius}
```

Comment:
Anastasiia asked me to make the example implementation in Dropdown (too) as SelectedItemBadge is expected to change, but Dropdown will need to support this as well.